.github/workflows/node_modules-publish.yml

```diff
@@ -65,14 +66,19 @@ jobs:
       - name: Stage git commit
         run: |
           set -ux
+          cat - artifact/npm-install.log > commit-message <<EOF
+          PR: $(sed 's@//api.@//@; s@/repos/@/@; s@/pulls/@/pull/@;' <<< "${PULL_URL}")
```

Comment:
Can you prefix this with `#` to make it clickable when viewing in github?

What's with the double `//`? If this is supposed to just show the number, why not just take the `.number` property instead of `.url`, or use `basename` ?

Reply:
This sed is about turning the https://api... URL into the normal human HTML URL.